system/jlib/jdebug.cpp

```diff
@@ -643,6 +643,56 @@ MODULE_EXIT()
 }
 
 
+//===========================================================================
+
+
+void BlockedTimeTracker::noteWaiting()
```

Comment:
is jtime.* a better place for these?

Reply:
I kind of agree with Jake since these classes are more than likely to be used by metrics to measure latency and not in a debug only mode.

Reply:
I'm not sure.

It is in jdebug because that is where the other classes for measuring times currently live - TimeSection, CycleTimer etc.

jdebug isn't a great name - since most (none?) of the functions don't have much to do with debugging.  They could all move to jtime, or create a new jtiming.hpp.  For the moment I would leave here.